docs/tezos-basics/economics_and_reward.md

```diff
@@ -0,0 +1,95 @@
+---
+id: economics_and_reward
+title: Economics and reward
+---
+
+
+## Transaction cost
+
+Each Tezos transaction has a cost. The user includes a gas fee when he submits his transaction. Bakers choose transactions with a minimal fee filter. If the baker chooses this transaction, he will add the operation to the block and will propagate it. These fees are used to pay the bakers. They are calculated with the following formula [1](https://opentezos.com/tezos-basics/economics_and_reward#references): 
+```
+fees >= (minimal_fees + minimal_nanotez_per_byte * size + minimal_nanotez_per_gas_unit * gas)
+```
+By default a transaction have this values:
+```
+minimal_fees = 0.000 1ꜩ (100µꜩ)
+minimal_nanotez_per_gas_unit = 100nꜩ/gu (0.000 000 1ꜩ/gu)
+minimal_nanotez_per_byte = 250nꜩ/B (0.000 00025ꜩ/B)
+```
+
+The size is the number of bytes of the complete serialized operation.
+
+In a Tezos transaction, we can add additional information such as smart contracts deployment, smart contract call, etc. This information has a storage cost which is represented with a `minimal_nanotez_per_byte`. Since the Delphy protocol update [2](https://opentezos.com/tezos-basics/economics_and_reward#references) this cost has been reduced from 1ꜩ to 0.25 ꜩ per kilobyte. The price to create a new account is thus lowered from 0.257 tez to 0.06425 tez (the cost to create an account is important to protect the system from spams and Sybil attacks [3](https://opentezos.com/tezos-basics/economics_and_reward#references)).
+
+On the Tezos blockchain, gas refers to the cost necessary to perform a transaction on the network. Bakers set the price of gas based on supply and demand for the computational power of the network needed to process smart contracts and other transactions.
```

Comment:
needed to process smart contracts and other transactions -> needed to process smart contract calls and other transactions